apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -135,6 +135,16 @@ class SettingsCog extends Component {
     return pageConstants && pageConstants.aiEnabled;
   }
 
+  levelbuilderModel() {
+    let model = {};
+    let pageConstants = getStore().getState().pageConstants;
+    if (pageConstants && pageConstants.aiModelId && pageConstants.aiModelName) {
```

Comment:
another good use-case for optional chaining! this conditional would become: `pageConstants?.aiModelId && pageConstants?.aiModelName`